src/catalog/tests/snapshots/debug__opened_trace.snap

```diff
@@ -374,7 +374,7 @@ Trace {
                                                 ),
                                                 replica_name: "r1",
                                                 logical_size: "1",
-                                                disk: false,
+                                                disk: true,
```

Comment:
Should these be changing? Was the behavior wrong before?

Reply:
This PR seems to have changed some defaults, I haven't actually checked if this is wanted!

Reply:
Undone